### PR TITLE
add/view transaction

### PR DESCRIPTION
**Pull request by haywhyz**
new transaction 
view transaction
view single transaction page

### CODE REVIEW

**Comment by Jeremiahiro** [comment]
pull from upstream then run composer install so you would resolve conflict on the composer.lock file

File: app/Http/Controllers/TransactionController.php [hunk 1]
```diff
@@ -24,27 +24,27 @@ public function __construct()
...
+          $headers = ['headers' => ['x-access-token' => Cookie::get('api_token')]];
+          $response = $client->request('GET', $url, $headers);
+          $statusCode = $response->getStatusCode();
+          if ($statusCode == 200) {
+              $body = $response->getBody()->getContents();
+              $transactions = json_decode($body)->data->details;
+            //   dd($transactions);
+              return view('backend.transaction.index')->with('response', $transactions);
+          }
+          if ($statusCode == 500) {
```
Comment (Jeremiahiro) [inline app/Http/Controllers/TransactionController.php:41]: always return or redirect to view with the exact error and not 500 error page, let this be our last option

File: resources/views/backend/transaction/index.blade.php [hunk 2]
```diff
@@ -94,20 +94,31 @@
...
+                        @foreach ($details->transactions as $transactions)
+                            <tr>
+                            <td>{{$transactions->_id }}</td>
+                             <td>{{$transactions->type }}</td>
+                            <td>{{$transactions->customer_ref_id }}</td>
+                            <td>{{$transactions->total_amount}}</td>
+                           
+                            {{-- <td>{{ date('d M Y', strtotime($transactions->created_date)) }}</td> --}}
+                            <td><a href="{{ route('transaction.show', $transactions->_id) }}" class="btn btn-primary btn-sm">view</i></a>
+                            <a href="{{ route('transaction.edit', $transactions->_id) }}" class="btn btn-success btn-sm">edit</i></a>
```
Comment (Jeremiahiro) [inline resources/views/backend/transaction/index.blade.php:113]: you delete does not have a confirmation option

**Comment by Jeremiahiro** [comment]
review 
then pull from upstream, run composer install afterward to resolve conlict

File: resources/views/backend/transaction/index.blade.php (hunk 2 above)
Comment (haywhyz) [reply resources/views/backend/transaction/index.blade.php:113]: based on the sheet , am not the one to implement that , but i can work on it also

File: app/Http/Controllers/TransactionController.php (hunk 1 above)
Comment (haywhyz) [reply app/Http/Controllers/TransactionController.php:41]: ok

**Pull request closed without merging**